parasolr/query/aliased_queryset.py

```diff
@@ -102,6 +102,13 @@ def highlight(self, field: str, **kwargs) -> "AliasedSolrQuerySet":
         field = self.field_aliases.get(field, field)
         return super().highlight(field, **kwargs)
 
+    def group(self, field: str, **kwargs) -> "AliasedSolrQuerySet":
+        """Extend :meth:`parasolr.query.queryset.SolrQuerySet.group`
+        to support using aliased field names in kwargs."""
+        field = self.field_aliases.get(field, field)
+        # TODO: should we also reverse alias for sort option if specified?
```

Comment:
Is this necessary / is there a need for it? Doesn't seem to be impacting us so far.

Reply:
I think it's a nice to have and not essential. It might be confusing if it doesn't get aliased but should be pretty easy to figure out. I can make an issue to track it and make a note in the docstring that only the group field is aliased.